### ISSUE TITLE
"external_url" property appears to be stripped from JSON feeds

### ISSUE THREAD

**Issue by nickheer** [feedbin/feedbin#647]
I am so glad that Feedbin supports JSON feeds. However, I am seeing a problem with the `external_url` property — specifically, Feedbin appears to be stripping it from the feed.

Background: each item in a JSON feed may offer an [`external_url` property](https://jsonfeed.org/version/1#items):

> `external_url` (very optional, string) is the URL of a page elsewhere. This is especially useful for linkblogs. If `url` links to where you’re talking about a thing, then `external_url` links to the thing you’re talking about.

Here's my feed and headers: https://gist.github.com/f66485073115b990ec0bafb1daec6c61

The relevant section of a sample item looks like this:

```json
"external_url" : "https://techcrunch.com/2021/07/28/calgarys-parking-authority-exposed-drivers-personal-data-and-tickets/",
"id" : "https://pxlnv.com/linklog/calgary-parking-security/",
"title" : "Calgary Parking Authority Exposed Drivers’ Personal Data and Tickets",
"url" : "https://pxlnv.com/linklog/calgary-parking-security/"
````

This means that, if I add my feed to NetNewsWire in my Mac-only feeds, the post title will go to the pxlnv.com… blog post, and it will also surface the techcrunch.com… URL that I am linking to:

![NetNewsWire screenshot](https://i.imgur.com/tfu23gM.png)

However, when I add JSON feeds to my Feedbin account, the techcrunch.com… URL is not visible in NetNewsWire. The line beginning "Link:" simply does not show up.

This happens in all similarly-formatted JSON feeds when using Feedbin as a syncing back-end.

The Feedbin ID of `https://pxlnv.com/feed/json` is `1759108`, if that helps. Please let me know if I can provide anything else to help track down this bug or resolve it.

**Comment by benubois** [comment]
Hi @nickheer,

Feedbin passes through JSON Feed data in its API response (see below).

I think this came up a while back on the NetNewsWire Slack, but maybe it never made it into a release. Either way I think [creating an issue](https://github.com/Ranchero-Software/NetNewsWire/issues) there could help move this forward.

Thanks!

```json
{
   "author" : "Nick Heer",
   "content" : "<p><a href=\"https://www.cnbc.com/2021/07/27/alphabet-googl-earnings-q2-2021.html\">Jennifer…",
   "created_at" : "2021-07-29T04:56:00.445547Z",
   "enclosure" : null,
   "extracted_articles" : [],
   "extracted_content_url" : "https://extract.feedbin.com/parser/feedbin/57b67f46122603d9a7b4a8ce022d9e8f0d67070e?base64_url=aHR0cHM6Ly9weGxudi5jb20vbGlua2xvZy9nb29nbGUtZmFjZWJvb2stZWFybmluZ3MtanVseS0yMDIxLw==",
   "feed_id" : 1759108,
   "id" : 3234353542,
   "images" : null,
   "json_feed" : {
      "author" : {
         "name" : "Nick Heer"
      },
      "authors" : [],
      "external_url" : "https://www.cnbc.com/2021/07/27/alphabet-googl-earnings-q2-2021.html",
      "id" : "https://pxlnv.com/linklog/google-facebook-earnings-july-2021/"
   },
   "original" : null,
   "published" : "2021-07-29T04:38:17.000000Z",
   "summary" : "Jennifer Elia…",
   "title" : "The Online Advertising Duopoly Reports Massive Earnings",
   "twitter_id" : null,
   "twitter_thread_ids" : [],
   "url" : "https://pxlnv.com/linklog/google-facebook-earnings-july-2021/"
}

**Comment by nickheer** [comment]
Ah, so sorry! Thanks @benubois
